### ISSUE TITLE
change URL to CP QA index page

### ISSUE THREAD

**Issue by kdionisio** [HumanExposure/factotum#915]
On the Chemical Presence QA index page (qa/chemicalpresence/), the external URL for each data group is being rendered as a relative, local URL.

Definition of Done:
external URL for each data group should have the http:// prefix before the {{datagroup.url}} tag

**Comment by zachmunzenrider** [comment]
With Kathie's ok, I went ahead and added URL validation to 3 model fields that didn't already have it: Datasource, Datagroup, and Datadocument. I also added a column to this table to display the URL link, and changed the hyperlinked name to point to the detail page instead.

Dave (or someone) ought to do a little data cleanup on prod to correct the small handful of records that don't currently have valid URLs (which includes the schema, e.g. "http://" in their URL fields. I checked my local copy of prod and found only 10 or 20 total that need to have this prepended to their current values.

select * from dashboard_datadocument where url not like 'http%' and url is not null and url > '';
select * from dashboard_datagroup where url not like 'http%' and url is not null and url > '';
select * from dashboard_datasource where url not like 'http%' and url is not null and url > '';
select * from dashboard_script where url not like 'http%' and url is not null and url > '';
